python_client/kubetorch/resources/compute/compute.py

```diff
@@ -1554,6 +1586,11 @@ def replicas(self, value: int):
                 }
             distributed_config["quorum_workers"] = value
             self.service_manager.set_replicas(self._manifest, value, distributed_config=distributed_config)
+
+            # Prevent cluster autoscaler from evicting pods during distributed training.
```

Comment:
should this be set outside of the `TrainJobServiceManager` check, to apply to any case where distributed_config is set (including knative and ray)

Reply:
then in the `distributed_config` setter should we only apply it in the train job case?